docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md

```diff
@@ -25,9 +24,12 @@ function getRestakeableStrategies() returns (address[] memory);
 }
 ```
 
-In order to have its name, information, and logo indexed, an AVS MUST invoke `updateAVSMetadataURI()` on the [AVSDirectory contract](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/core/AVSDirectory.sol).
+New AVS Listings: in order for an AVS to have its name, information, and logo indexed, it must invoke `updateAVSMetadataURI()` on the [AVSDirectory contract](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/core/AVSDirectory.sol).
 It currently takes about 10 minutes for it to be indexed and the metadata to be updated on the dashboard.
 
+Updating AVS Listings:  if you deploy a new contract for a new version of your AVS, please be sure to remove the previous listing. Invoke the update metadata function with value of null, such as `updateAVSMetadataURI("")` to remove the previous listing. Your listing will then be removed from the application cache within one hour.
```

Comment:
maybe a bit more clear to word like this?

`Updating AVS Listings: When you deploy a new contract for a new version of your AVS, make sure to remove the previous listing by invoking the updateAVSMetadataURI function with a null value, like this: updateAVSMetadataURI(""). This will remove your previous listing from the application cache within one hour.`